src/components/SkipLink/SkipLink.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+/**
+ * This is a [React](https://reactjs.org/) component for the Vanilla [Skip link](https://vanillaframework.io/docs/patterns/links#skip-link) component.
+ */
+export const SkipLink = (): JSX.Element => {
```

Comment:
I think `href` should be a prop. The detail that the `href` reference must link up with the id of `AppMain`, and it has to be `main-content` is currently hidden inside this component. Having `href` would help especially In case if a downstream project decides to not use `ApplicationLayout` and use `AppMain` directly.

Reply:
I see your point about customising the `href`, and it is a valid one indeed. I will keep `main-content` as a default though, since that's the standard that we use across projects.

If you use `AppMain` directly, you need to add `SkipLink` manually to your application. Currently, the only way to have it "for free" is by using `ApplicationLayout`, as it needs to be placed in a certain position of the layout to work. So if you don't use `ApplicationLayout` but only `AppMain`, you fall in the manual/customised case.

Reply:
yeah having a default makes sense :+1: